scripts/check-cxx11abi.sh

```diff
@@ -13,7 +13,7 @@ fi
 LIBSTDCPP=$(ldd $(dirname $0)/check-cxx11abi.dat |grep libstdc++ |cut -d' ' -f3)
```

Comment:
I think now that we inverted the semantics of this script, we should rename it to `check-cxx03abi`